segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/security/TLSConfigChangeEventConsumerTests.java

```diff
@@ -42,8 +42,8 @@ public void testEmptyPathToKeyFileIsRejected() {
 
     @Test
     public void testInvocationIncrementsReloadCounter() {
-        String pathToCertificateFile = "../../config/" + SecurityConfigDefaults.TLS_SERVER_CERT_FILE_NAME;
-        String pathToKeyFile = "../../config/" + SecurityConfigDefaults.TLS_SERVER_PRIVATE_KEY_FILE_NAME;
+        String pathToCertificateFile = "../../../config/" + SecurityConfigDefaults.TLS_SERVER_CERT_FILE_NAME;
+        String pathToKeyFile = "../../../config/" + SecurityConfigDefaults.TLS_SERVER_PRIVATE_KEY_FILE_NAME;
```

Comment:
This part is replicated multiple times and also contains magic ../../... 
While we are at it, looks like a good candidate to move into one class where all test const/configs are defined.

Reply:
I did not write these things. This is just to correct a huge PR in which somehow these classes got lost in the wrong package. I'll leave it to @ravisharda to fix these tests properly.